lib/src/features/event/view/event_view.dart

```diff
@@ -1,26 +1,110 @@
+import 'dart:math';
+
 import 'package:flutter/material.dart';
-import 'package:myapp/src/router/coordinator.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:myapp/src/features/event/logic/event_view_bloc.dart';
+import 'package:myapp/src/features/event/logic/event_view_state.dart';
+import 'package:myapp/src/features/event/widget/calendar_event_widget.dart';
+import 'package:myapp/src/features/event/widget/category_event_widget.dart';
+import 'package:myapp/src/features/event/widget/map_page.dart';
+import 'package:myapp/src/features/home/logic/event_item_bloc.dart';
+import 'package:myapp/src/features/home/widget/event_item_home.dart';
+import 'package:myapp/src/theme/colors.dart';
+
+import 'package:myapp/widgets/appbar/app_bar_custom.dart';
 
 class EventHomeView extends StatelessWidget {
   const EventHomeView({super.key});
 
   @override
   Widget build(BuildContext context) {
-    return Scaffold(
-      appBar: AppBar(
-        title: const Text("Event screen"),
-      ),
-      body: Center(
-        child: Column(
-          children: [
-            ElevatedButton(
+    return BlocProvider(
+      create: (_) => EventViewBloc()..updateWeekDays(DateTime.now()),
+      child: const EventHomePage(),
+    );
+  }
+}
+
+class EventHomePage extends StatelessWidget {
+  const EventHomePage({
+    super.key,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return BlocBuilder<EventViewBloc, EventViewState>(
+      buildWhen: (previousState, currentState) {
+        return previousState != currentState;
```

Comment:
Change buildWhen for all code in the app, must use the param in state to check, not the entire state